{
  try {
    cache("LON",0).put("k","v");
    fail("This should have thrown an exception");
  }
 catch (  Exception e) {
  }
}
